data-science-onramp/ai-platform/modules/trainer/tfkeras_model/task.py

```diff
@@ -141,17 +147,14 @@ def train_and_evaluate(
     )
 
     # Export model
-    export_path = os.path.join(job_dir, "tfkeras_model/")
-    tf.keras.models.save_model(keras_model, export_path)
-    print(f"Model exported to: {export_path}")
+    keras_model.save(model_dir)
```

Comment:
Are you sure this works for training on the cloud?

I had to explicitly use `tf.keras.models.save_model` because it supports saving to a GCS bucket, whereas `keras_model.save` only works locally.

Reply:
For reference, I came across this issue following this tutorial: https://cloud.google.com/ai-platform/docs/getting-started-keras

Specifically, this command:

```
gcloud ai-platform jobs submit training $JOB_NAME \
  --package-path trainer/ \
  --module-name trainer.task \
  --region $REGION \
  --python-version 3.7 \
  --runtime-version 1.15 \
  --job-dir $JOB_DIR \
  --stream-logs
  ```

Reply:
Yep! The test checks that the file was written to a GCS bucket, so these wouldn't pass if this didn't work.

Reply:
Interesting, I wonder if uCAIP copies it over for you or if I'm not understanding `keras_model.save`.

Good to know, thanks!